### DISCUSSION TITLE
Enabling contribution of collections

### DISCUSSION THREAD

**Discussion by kameshbhr** [Sunbird-coKreat/Community#47] [Q&A]
Is it possible to configure a Collection category as one of the Content types in the Sourcing Project scope, so that we can enable contribution of Collections apart from individual contents?

**Comment by vinukumar-vs** [comment]
@kameshbhr To enable contribution of collection requires a code change on CoKreat portal. Also presently we dont have collection publish API's(Auto creator job) from CoKreat to consumption portal(sunbird-ED). 
This requires knowlg team support to onboard collection publish. We have the roadmap item for this.

@pallakartheekreddy @luckynvdu

**Reply by kameshbhr** [reply]
Thanks @vinukumar-vs 
Can you please share the link to the product backlog ticket for this, for traceability purpose. I would like to know when is it likely to be picked up.

**Reply by Krishnaj20** [reply]
@kameshbhr - can you explain what use cases you are thinking of by enabling this capability please?

**Reply by kameshbhr** [reply]
@Krishnaj20 
1. Being able to source collections (like playlists) apart from individual content pieces from external contributors
2. Creating Textbooks, Courses etc. using Sourcing portal. This will help manage the creation and publishing in more controlled manner. For example, if a state has multiple teams to create and publish multiple types of courses, and wants to control that reviewers of one team cannot review and publish courses of other team, they will be able to do it using the sourcing projects. This is not possible with the current Workspace.

We have got both of these requirements in DIKSHA.
